funding-laundering-detector-py/src/agent.py

```diff
@@ -345,13 +390,17 @@ async def analyze_transaction(transaction_event: forta_agent.transaction_event.T
                                                               labels, DENOMINATOR_COUNT_FUNDING, CHAIN_ID))
 
         # LAUNDERING
-        elif to in confirmed_targets_keys and from_ not in confirmed_targets_keys:
+        elif to in confirmed_targets_keys and from_ not in confirmed_targets_keys or to in mixer_addresses.get(CHAIN_ID, []):
             DENOMINATOR_COUNT_LAUNDERING += 1
+            if to in mixer_addresses.get(CHAIN_ID, []):
+                label = "mixer"
+            else:
+                label = confirmed_targets[to]['type']
             usd, token = calculate_usd_for_base_token(value, CHAIN_ID)
             if usd > thresholds["TRANSFER_THRESHOLD_IN_USD"] and (
                     confirmed_targets[to]['type'] != 'dex' or not DEX_DISABLE) and (
```

Comment:
```suggestion
            if usd > thresholds["TRANSFER_THRESHOLD_IN_USD"] and (
                    (to in confirmed_targets and confirmed_targets[to]['type'] != 'dex') or not DEX_DISABLE) and (
```

Reply:
`to in confirmed_targets` will not be met if it's just manually caught from the mixers 🤔